spot_wrapper/wrapper.py

```diff
@@ -908,6 +909,14 @@ def __init__(
             self._estop_monitor,
         ]
 
+        self._spot_docking = SpotDocking(
+            self._robot,
+            self._logger,
+            self._state,
+            self._docking_client,
```

Comment:
Missing RobotCommandData

Reply:
Otherwise with the cherry pick of PR #58, the above fix and the added import of `from spot_wrapper.wrapper_helpers import RobotCommandData, RobotState`  into the spot_docking.py seems to work with the appropriate changes to spot_ros2. @heuristicus Go head and make the above fixes and put #58 up for review. Once its all together I can do another quick test to make sure it all works.